service/matching/matchingEngine.go

```diff
@@ -234,14 +237,19 @@ func (e *matchingEngineImpl) AddDecisionTask(ctx context.Context, addRequest *ma
 		return false, err
 	}
 
-	taskInfo := &persistence.TaskInfo{
-		DomainID:               domainID,
-		RunID:                  addRequest.Execution.GetRunId(),
-		WorkflowID:             addRequest.Execution.GetWorkflowId(),
-		ScheduleID:             addRequest.GetScheduleId(),
-		ScheduleToStartTimeout: addRequest.GetScheduleToStartTimeoutSeconds(),
-		CreatedTime:            time.Now(),
+	// This needs to move to history see - https://github.com/temporalio/temporal/issues/181
+	now := types.TimestampNow()
+	expiry := types.TimestampNow()
```

Comment:
This is new behavior as we no longer have access to ScheduleToStartSeconds here. Only a timestamp we need to reintrepret, see  https://github.com/temporalio/temporal/issues/181